Cargo.toml

```diff
@@ -526,3 +527,17 @@ wasmi = { opt-level = 3 }
 x25519-dalek = { opt-level = 3 }
 yamux = { opt-level = 3 }
 zeroize = { opt-level = 3 }
+
+# [patch."https://github.com/w3f/ring-vrf"]
+# bandersnatch_vrfs = { path = "/mnt/ssd/develop/w3f/ring-vrf/bandersnatch_vrfs" }
+# bandersnatch_vrfs = { git = "https://github.com/davxy/ring-vrf", branch = "davxy-nitpicks" }
+
+# [patch."https://github.com/w3f/ring-proof"]
+# ring = { path = "/mnt/ssd/develop/w3f/ring-proof/ring" }
+# common = { path = "/mnt/ssd/develop/w3f/ring-proof/common" }
+# ring = { git = "https://github.com/davxy/ring-proof", branch = "davxy-nitpicks" }
+# common = { git = "https://github.com/davxy/ring-proof", branch = "davxy-nitpicks" }
+
+[patch."https://github.com/w3f/fflonk"]
+fflonk = { git = "https://github.com/davxy/fflonk", branch = "davxy-patch" }
```

Comment:
Waiting for:
- https://github.com/w3f/fflonk/pull/37
- https://github.com/w3f/fflonk/pull/36